### DISCUSSION TITLE
Included SD Card Worked For Me

### DISCUSSION THREAD

**Discussion by zeulewan** [bassamanator/Sovol-SV06-firmware#164] [General]
I used the 8GB KingFast SD card that came with the printer formatted as FAT32 with MacOS and it worked. I have a 32GB and 128GB SanDisk SD card and neither of them worked. I thought I would just make a post to potentially help others in the future. I'm not sure why the 32GB or 128GB didn't work, there might be a size limit on the SD card. If anyone wants to reply with the SD card that worked for them please do.

**Comment by bassamanator** [comment]
Only some users have flashing issues with the Sovol microSD card. Moreover, as stated, 16GB is the hard limit.
